### PR TITLE
PRKT-88 Rename Serial Driver

### PR DESCRIPTION
**Pull request by BryceVichiOJ**
### Deprecated
- Deprecated class mechaspin::parakeet::Driver, replaced by mechaspin::parakeet::Pro::Driver

### CODE REVIEW

File: src/parakeet/Driver.cpp [hunk 1]
```diff
@@ -2,567 +2,4 @@
...
-    {
-        if(!serialPort.isConnected())
-        {
-            throw exceptions::NotConnectedToSensorException();
-        }
-    }
-}
-}
\ No newline at end of file
+#include <parakeet/Driver.h>
```
Comment (nichmj01) [inline src/parakeet/Driver.cpp]: Seems like we don't need this file at all now?

File: include/parakeet/Driver.h [hunk 2]
```diff
@@ -2,173 +2,17 @@
...
-        std::chrono::milliseconds interfaceThreadStartTime;
-        int interfaceThreadFrameCount = 0;
-        std::thread serialInterfaceThread;
-        bool runSerialInterfaceThread;
-
-        std::chrono::time_point<std::chrono::system_clock> timeOfFirstPoint;
-        std::vector<PointPolar> pointHoldingList;
-        std::function<void(const ScanDataPolar&)> scanCallbackFunction = nullptr;
-};
+    PARAKEET_DEPRECATED(typedef Pro::Driver Driver);
```
Comment (nichmj01) [inline include/parakeet/Driver.h:15]: To verify cause I've never done it before; if I instantiate mechaspin::parakeet::Driver I'll get a deprecated warning message from the compiler?

Comment (BryceVichiOJ) [reply include/parakeet/Driver.h:15]: Yep. Tested it, as I was questioning the same thing.

File: src/parakeet/Driver.cpp (hunk 1 above)
Comment (BryceVichiOJ) [reply src/parakeet/Driver.cpp]: I didn't like the idea of removing files. Realistically, only header files should not be removed, as they can be directly interacted with. Cpp's are fine to remove. I'll get on that now.

**Review by nichmj01 — APPROVED** [review]

**Pull request merged**